apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -758,24 +758,7 @@ const (
 	// Its condition status can be one of the following:
 	// - "True" means we have successfully scheduled the resources to satisfy the placement requirement.
 	// - "False" means we didn't fully satisfy the placement requirement. We will fill the Message field.
-	// TODO, use "Scheduled" instead.
-	ResourceScheduledConditionType ResourcePlacementConditionType = "ResourceScheduled"
```

Comment:
how come no existing code use this?

Reply:
These conditions are info only and validated by tests.

To fix the rp side e2e tests, we just need to upgrade the fleet go mod.
https://msazure.visualstudio.com/CloudNativeCompute/_search?action=contents&text=place_resources_with_pickfixed_crp&type=code&lp=code-Project&filters=ProjectFilters%7BCloudNativeCompute%7DRepositoryFilters%7Baks-rp%7D&pageSize=25&result=DefaultCollection/CloudNativeCompute/aks-rp/GBmaster//test/e2e/suites/pkg/handlerk8s/handler_fleet_workload.go

or we can leave there and fix the rp e2e tests when enabling the new conditions.

i'm fine with either way.